rockcraft/extensions/expressjs.py

```diff
@@ -114,15 +114,15 @@ def _check_project(self) -> None:
             or "start" not in self._app_package_json["scripts"]
         ):
             raise ExtensionError(
-                "missing start script",
+                "missing start script in package.json",
                 doc_slug="/reference/extensions/express-framework",
                 logpath_report=False,
             )
         if "name" not in self._app_package_json or not isinstance(
             self._app_package_json["name"], str
         ):
             raise ExtensionError(
-                "missing application name",
+                "missing application name in package.json",
```

Comment:
Similar thought here:


```suggestion
                "missing 'name' field in package.json",
```